### PR TITLE
Navigate To Thank You Page On Successful Late Pledge

### PR DESCRIPTION
**Pull request by scottkicks**
# 📲 What

Navigates To Thank You Page On Successful Late Pledge

# 🤔 Why

So we can formally thank users for their pledge, show them verification, and suggest other projects to them

# 🛠 How

Using data we already have in the PostCampaignCheckoutViewModel, gather the data needed for the Thanks Page.

# 👀 See

I can't remove backings rn and only have 2 accounts that I used to test already so I can't screen record this.

# ✅ Acceptance criteria

- [x] On successful backing, users are navigated to the Thanks Page which displays [the appropriate info ](https://www.figma.com/file/sFfDKxlJ2tiuq1xgIsaoiI/Late-Campaign-Backings?type=design&node-id=54-5016&mode=design&t=oQ3DObf65PfyXZKz-0)

### CODE REVIEW

File: Library/ViewModels/PostCampaignCheckoutViewModel.swift [hunk 1]
```diff
@@ -369,7 +370,17 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
         AppEnvironment.current.apiService.completeOnSessionCheckout(input: input).materialize()
       }
 
-    self.checkoutComplete = checkoutCompleteSignal.signal.values().ignoreValues()
+    let thanksPageData = Signal.combineLatest(initialData, baseReward)
+      .map { initialData, baseReward -> ThanksPageData? in
+        guard let reward = baseReward else { return nil }
+
+        return (initialData.project, reward, nil, initialData.total)
+      }
```
Comment (scottkicks) [inline Library/ViewModels/PostCampaignCheckoutViewModel.swift:378]: It didn't look like we needed to pass any CheckoutData in this case so I passed nil instead.

Comment (amy-at-kickstarter) [reply Library/ViewModels/PostCampaignCheckoutViewModel.swift:378]: Huh. Do regular pledges require it, or can `CheckoutData` be cleaned up from the `ThanksPageData` type?

**Review by amy-at-kickstarter — APPROVED** [review]
LGTM, one suggestion if it's possible.

File: Library/ViewModels/PostCampaignCheckoutViewModel.swift (hunk 1 above)
Comment (scottkicks) [reply Library/ViewModels/PostCampaignCheckoutViewModel.swift:378]: Yea I have a note to look at that. I couldn't see find what it was needed for honestly.

**Pull request merged**